Formula/b/bluez.rb

```diff
@@ -0,0 +1,32 @@
+class Bluez < Formula
+  desc "Bluetooth protocol stack for Linux"
+  homepage "https://github.com/bluez/bluez"
+  url "https://mirrors.edge.kernel.org/pub/linux/bluetooth/bluez-5.72.tar.xz"
+  sha256 "499d7fa345a996c1bb650f5c6749e1d929111fa6ece0be0e98687fee6124536e"
+  license "GPL-2.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
```

Comment:
Do we still need these given the tarball switch?

Reply:
yeah, that is right, let me remove them.